src/util/mp_arith.cpp

```diff
@@ -315,3 +319,253 @@ unsigned integer2unsigned(const mp_integer &n)
   assert(ull <= std::numeric_limits<unsigned>::max());
   return (unsigned)ull;
 }
+
+/*******************************************************************\
+
+Function: bitwise_or
+
+  Inputs:
+
+ Outputs:
+
+ Purpose: bitwise or
+          bitwise operations only make sense on native objects, hence the
+          largest object size should be the largest available c++ integer
+          size (currently long long)
+
+\*******************************************************************/
+
+mp_integer bitwise_or(const mp_integer &a, const mp_integer &b)
+{
+  ullong_t result=a.to_ulong()|b.to_ulong();
```

Comment:
there's an internal use for them in the interpreter, where they will be applied to C/Java standard types only